internal/cli/atlas_clusters_create_test.go

```diff
@@ -28,27 +29,83 @@ func TestAtlasClustersCreate_Run(t *testing.T) {
 
 	defer ctrl.Finish()
 
-	expected := fixtures.Cluster()
-
-	createOpts := &atlasClustersCreateOpts{
-		globalOpts:   newGlobalOpts(),
-		name:         "ProjectBar",
-		region:       "US",
-		instanceSize: atlasM2,
-		members:      3,
-		diskSizeGB:   10,
-		backup:       false,
-		mdbVersion:   currentMDBVersion,
-		store:        mockStore,
-	}
-
-	mockStore.
-		EXPECT().
-		CreateCluster(createOpts.newCluster()).Return(expected, nil).
-		Times(1)
-
-	err := createOpts.Run()
-	if err != nil {
-		t.Fatalf("Run() unexpected error: %v", err)
-	}
+	t.Run("flags run", func(t *testing.T) {
+		expected := fixtures.Cluster()
+
+		createOpts := &atlasClustersCreateOpts{
+			globalOpts:   newGlobalOpts(),
+			name:         "ProjectBar",
+			region:       "US",
+			instanceSize: atlasM2,
+			members:      3,
+			diskSizeGB:   10,
+			backup:       false,
+			mdbVersion:   currentMDBVersion,
+			store:        mockStore,
+		}
+
+		cluster, _ := createOpts.newCluster()
+		mockStore.
+			EXPECT().
+			CreateCluster(cluster).Return(expected, nil).
+			Times(1)
+
+		err := createOpts.Run()
+		if err != nil {
+			t.Fatalf("Run() unexpected error: %v", err)
+		}
+	})
+
+	t.Run("file run", func(t *testing.T) {
+		appFS := afero.NewMemMapFs()
+		// create test file
+		fileYML := `
+{
+  "name": "ProjectBar",
```

Comment:
[q] Wondering should we put this is a samples or tests folder and load them in a loop - would make it easier to extend in the future.

Reply:
We are using `afero`, afero has a feature to read/write from memory (`afero.NewMemMapFs()`), it does not write to disk and this is ideal for unit tests as they should be faster and guarantee no disk pollution
I did discussed with @andreaangiolillo maybe adding it to the e2e tests... maybe

Reply:
Excellent. Sounds good.